fbrp/src/fbrp/runtime/conda.py

```diff
@@ -81,16 +81,18 @@ async def run(self):
 
         # We grab the conda env variables separate from executing the run
         # command to simplify detecting pid and removing some race conditions.
-
+        subprocess_env = os.environ
```

Comment:
```py
subprocess_env = os.environ.copy()
```
Otherwise, you're making a shallow copy and the update will modify the global `os.environ`
https://stackoverflow.com/questions/2231227/python-subprocess-popen-with-a-modified-environment#answer-4453495